kani-driver/src/call_goto_instrument.rs

```diff
@@ -183,6 +192,18 @@ impl KaniSession {
         self.call_goto_instrument(&args)
     }
 
+    /// Apply annotated loop contracts.
+    pub fn instrument_loop_contracts(&self, file: &Path) -> Result<()> {
+        let args: Vec<OsString> = vec![
+            "--apply-loop-contracts".into(),
+            "--loop-contracts-no-unwind".into(),
+            "--disable-loop-contracts-side-effect-check".into(),
```

Comment:
Why is that?

Reply:
Because loop contracts now are wrapped in a closure which will be a side-effect expression in CBMC even they may not contain side-effect. So we disable the side-effect check for now and will implement a better check instead of simply rejecting function calls and statement expressions.
https://github.com/diffblue/cbmc/issues/8393

Reply:
Can you please link this issue here with your explanation?